### DISCUSSION TITLE
Theme customization limitation?

### DISCUSSION THREAD

**Discussion by Sassiest01** [fief-dev/fief#364] [Q&A] [answered]
I have created a new theme so I could start customizing it but noticed the "Default text is still there. Is there a way for me to remove that text or modify what is says? There doesn't seem to be anything I can find, despite the docs showing different text then what I have, it didn't show a way for it to be modified. 

![image](https://github.com/fief-dev/fief/assets/71369310/fb767423-4374-4dd6-b834-7678e61c8e72)

I read in a post on Polar that "Advanced authentication pages customization" may possibly be added to Stronghold in the future (don't know if this is related to supplying jinja templates or not) but it would be nice to modify or remove text using the current theme modification

**Comment by fief-bailiff** [comment]
Hail, @Sassiest01 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to [star our repository](https://github.com/fief-dev/fief) ⭐️

<p align="center">
<img width="656" alt="star-fief" src="https://github.com/fief-dev/fief/assets/1144727/b2db7f59-102f-47e5-9751-67fa10372c52">
</p>

## Want to support us?

Subscribe to one of our paid plan to help us continue our work and receive exclusive information and benefits! Starts at $5/month 🪙

<p align="center">
    <a href="https://polar.sh/fief-dev">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/embed/subscribe.svg?org=fief-dev&darkmode=1">
      <img alt="Subscribe" src="https://polar.sh/embed/subscribe.svg?org=fief-dev">
    </picture>
    </a>
</p>

Farewell!

**Comment by frankie567** [comment]
Indeed, the current theme customization is limited to colors and font; the content can't be modified.

We're currently considering a feature where developers could indeed provide their own Jinja templates to override the content and make it home 🙂 I don't have an ETA on this but I'll post advancements in the following issue: https://github.com/fief-dev/fief/issues/352

**Reply by Sassiest01** [reply]
Thank you for your response. Is there a particular reason why "Default" is on that page? I can understand having Fief branding but not that text, at least in a non modifiable state.

I will keep an eye out on that issue, thank you.

**Reply by frankie567** [reply, marked as answer]
Ah :D "Default" is the name of your Tenant, created by default. You can change the name or even upload a logo, it'll appear over there: https://docs.fief.dev/configure/tenants/

**Reply by Sassiest01** [reply]
Thank you, I think I had 2 Tennant's assigned to the same theme and thus picked the name of the first Tennant. I will verify this when I am able to, thank you.

Edit: I may have copied the theme and changed the name assuming it would change based on its own name rather then the tenant using it.
